amarna/command_line.py

```diff
@@ -96,6 +96,7 @@ def main() -> int:
         metavar="-f",
         type=str,
         help="the name of the .cairo file or directory with .cairo files to analyze",
+        nargs="?"
     )
 
     parser.add_argument("-p", "--print", action="store_true", help="print output")
```

Comment:
Hi @LucasLvy, thanks for the PR! 
If the filename is optional, it should be checked that it is not None, in line 153.
And I just noticed that there is no check to validate if the provided filename exists. 
You can also add that check with `os.path.exists` around line 156